### ISSUE TITLE
<Router> children need keys?

### ISSUE THREAD

**Issue by kurtextrem** [developit/preact-router#230]
`preact/debug` tells me the following:

`Following component has two or more children with the same "key" attribute. This may cause glitches and misbehavior in rendering process. Component: <Router history="[object Object]" />`

Although I'm not sure if this is either an issue of `preact/debug` or `preact-router`, as I get the error for the following code too (unrelated to preact-router):
```jsx
<header className="media align-items-center card-block grow-0 pl-2 pr-2">
	<a href={`https://www.instagram.com/${user.username}`} target="_blank" rel="noopener">
		<img src={updateCDN(user.profile_pic_url)} className="img-fluid profile-pic rounded mr-2" />
	</a>
	<Media body>
		<a href={`https://instagram.com/${user.username}`} target="_blank" rel="noopener">
			{user.full_name || user.username}
		</a>
	</Media>
	<a href={`https://www.instagram.com/p/${props.code}`} target="_blank" rel="noopener">
		<TimeAgo className="text-muted" date={state.date} />
	</a>
</header>
```
Msg: `Following component has two or more children with the same "key" attribute. This may cause glitches and misbehavior in rendering process. Component: <header className="media align-items-center card-block grow-0 pl-2 pr-2"`

I don't think it would be my task to add keys here, is it?

**Comment by developit** [comment]
Hi @kurtextrem - sorry for the slow reply there.  If you're using a version of Preact prior to 8.2.4, `preact/debug` would incorrectly warn about duplicate keys when using unkeyed elements. The warning was fixed in 8.2.4.

**Comment by kurtextrem** [comment]
Thank you Jason, I assumed so.
